### ISSUE TITLE
Add data-update step to CASA base image

### ISSUE THREAD

**Issue by bennahugo** [SpheMakh/Stimela#276]
Should add the following step to casa base image, to allow users to rebuild casa docker image when the ephemeris gets out of date. CASA v5 raises SEVERE warnings when this is not kept in sync which induces artificial crashes. However we cannot not check for SEVERE log messages since this is the only way to check if there was a problem running a particular task

```
2) Without starting CASA one can update the repository from the Unix prompt, too:


# in a terminal
casa-config --exec update-data
```

**Comment by bennahugo** [comment]
Just to clarify:
 - add this line to the casa base docker file
 - docker build -t stimela/casa:[your version]
 - rebuild your cabs - stimela build

**Comment by bennahugo** [comment]
This may be necessary to do manually every few months as the leapsecond table gets out of date

**Comment by SpheMakh** [comment]
Fixed #284
